camayoc/tests/qpc/cli/utils.py

```diff
@@ -213,11 +183,8 @@ def cred_show_and_check(options, output, exitstatus=0):
 report_merge = functools.partial(cli_command, "{} -v report merge".format(client_cmd))
 """Run ``qpc report merge`` with ``options`` and return output."""
 
-
-def job_status(options=None, exitstatus=0):
-    """Run ``qpc scan job`` with ``options`` and return output."""
-    output = cli_command("{} -v scan job".format(client_cmd), options, exitstatus)
-    return json.loads(output)
+report_upload = functools.partial(cli_command, "{} -v report upload".format(client_cmd))
+"""Run ``qpc report merge`` with ``options`` and return output."""
```

Comment:
Variable and command are `report upload`, but docstring is `report merge`. I think docstring is wrong here.